nservicebus/sqlserver/configuration.md

```diff
@@ -10,6 +10,8 @@ redirects:
 
 ## Connection strings
 
+NOTE: SQL Server uses ADO.NET and as a consequence connection pooling mechanism built into it. This may result in connection pool being shared by transport and other parts of endpoint process. Depending on the situation it might be necessary to adjust the default connection pool size. More information about connection pooling and connection pool configuration can be found [here](https://msdn.microsoft.com/en-us/library/8xx3tyca.aspx).
```

Comment:
SQL Server transport uses...

Maybe the sentence is worth rephrasing:

The SQL Server transport is built on top of ADO.NET and uses its connection pooling mechanism.